Sources/Sentry/SentryOutOfMemoryTracker.m

```diff
@@ -48,6 +48,53 @@ - (instancetype)initWithOptions:(SentryOptions *)options
     return self;
 }
 
+- (NSArray *)loadBreadcrumbs
```

Comment:
h: I'm not a fan of this in here.
I think we should try as much as possible to apply "Separation of concerns", SentryOutOfMemoryTracker shouldn't deal with files or building breadcrumbs lists.

The SDK is already extremely tightly coupled as it is, we should avoid to increase coupling.

Reply:
Good point, I moved this logic to SentryFileManager